examples/cryptocurrency-advanced/backend/Cargo.toml

```diff
@@ -32,3 +32,7 @@ hex = "=0.3.2"
 
 [build-dependencies]
 exonum-build = { version = "0.9.0", path = "../../../components/build" }
+
+[features]
+default = ["with-serde"]
```

Comment:
Sorry for being stupid: what does this feature influence?

Reply:
rust-protobuf generates structs with serde support with the following annotations:
`#[cfg_attr(feature = "with-serde", derive(Serialize, Deserialize))]`

Reply:
Maybe we could file an issue to _rust-protobuf_ to stop throwing feature-gates like this into _generated_ code. I'd find it more natural to have some boolean flag in protoc invocation which will tell protoc whether to add `#[derive]` or not. Or have a feature on rust-protobuf itself which enables this everywhere. I'm definitely not a fan of having to add default features to our crates for this.